### DISCUSSION TITLE
helm keda-2.12.0 update Error: UPGRADE FAILED:

### DISCUSSION THREAD

**Discussion by safetylab** [kedacore/keda#5181] [Q&A / Need Help]
### Expected Behavior

I have hpa.yaml
---
apiVersion: keda.sh/v1alpha1
kind: ScaledObject
metadata:
  name: test
  namespace: test
  labels:
    deploymentName: test
spec:
  scaleTargetRef:
    name: test
  pollingInterval: 1
  cooldownPeriod: 30
  minReplicaCount: 3
  maxReplicaCount: 100
  triggers:
    - type: prometheus
      metadata:
        serverAddress: http://prometheus-kube-prometheus-prometheus.monitoring.svc.cluster.local:9090
        metricName: php_active_processes
        threshold: 10
        query: sum(php_active_processes{namespace="test"})

### Actual Behavior

When I install by helm upgrade --install - everything is ok.
But if I run it again (I changed   minReplicaCount or  maxReplicaCount)- update doesnt work - Error: UPGRADE FAILED: cannot patch "test" with kind ScaledObject: Timeout: request did not complete within requested timeout - context deadline exceeded && cannot patch "test" with kind ScaledObject:

### Steps to Reproduce the Problem

helm upgrade --install  2 times for my example

### Logs from KEDA operator

```
example
```

### KEDA Version

2.12.0

### Kubernetes Version

1.27

### Platform

Google Cloud

**Comment by tomkerkhove** [comment]
Can you share logs of new KEDA installation during please? There is not other way to investigate this
